### PR TITLE
Upgrade to V.5.6

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
- *New version upstream*

## Solution
- *Upgrade to version 5.6*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** :  
*Code review and approval have to be from a member of @YunoHost-Apps/apps-group*
- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/wordpress_ynh%20PR115%20(ericgaspar)/)

### CODE REVIEW

File: scripts/restore [hunk 1]
```diff
@@ -96,10 +96,12 @@ chown root: $final_path/wp-config.php
 
 ynh_restore_file --origin_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"
 
+ynh_add_fpm_config --package="$extra_php_dependencies"
```
Comment (kay0u) [inline scripts/restore]:
```suggestion
ynh_add_fpm_config --usage=$fpm_usage --footprint=$fpm_footprint --package="$extra_php_dependencies"
```

You must retrieve  `fpm_usage` and `fpm_footprint` in the `LOAD SETTINGS` section

File: conf/php-fpm.conf [hunk 2]
```diff
@@ -0,0 +1,430 @@
+; Start a new pool named 'www'.
```
Comment (kay0u) [inline conf/php-fpm.conf]: Why did you remove `conf/extra_php-fpm.conf ` and add back the template?

When we use `usage` and `footprint` parameters, like in the install script:
```
 ynh_add_fpm_config --usage=$usage --footprint=medium --package="$extra_php_dependencies"
```
we don't need this template, it's fully generated from the [default one](https://github.com/YunoHost/yunohost/blob/1279755463100e6c4a0ba7d4b50f8f39658c8eac/data/helpers.d/php#L165-L214)

**Review by kay0u — COMMENTED** [review]
Thank you for the work!

File: conf/php-fpm.conf (hunk 2 above)
Comment (ericgaspar) [reply conf/php-fpm.conf]: I can' recall the exact alert but with `conf/extra_php-fpm.conf`,  php7.3 service was failling and with the template it was working...
I'll give it another try and report the exact output error.

Comment (ericgaspar) [reply conf/php-fpm.conf]: well... I can't reproduce my "php7.3 service was failling" error...

File: scripts/restore [hunk 3]
```diff
@@ -94,12 +97,12 @@ chown root: $final_path/wp-config.php
 # RESTORE THE PHP-FPM CONFIGURATION
 #=================================================
 
-ynh_restore_file --origin_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"
+ynh_add_fpm_config --usage=$fpm_usage --footprint=$fpm_footprint --package="$extra_php_dependencies"
```
Comment (kay0u) [inline scripts/restore:104]: after rethinking of it... it'd be better if we use 
```
ynh_restore_file --origin_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"
```
to restore the potentially modified php conf, and use `ynh_add_fpm_config` right after which will make a diff.
so:
```suggestion
# Restore the file first, so it can have a backup if different
ynh_restore_file --origin_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"

# Recreate a dedicated php-fpm config
ynh_add_fpm_config --usage=$fpm_usage --footprint=$fpm_footprint --package="$extra_php_dependencies"
```
(Sorry for my shitty suggestion)

File: manifest.json [hunk 4]
```diff
@@ -18,7 +18,7 @@
 		"email": "maniackc_dev@crudelis.fr"
 	}],
     "requirements": {
-        "yunohost": ">= 3.8.1"
+        "yunohost": ">= 4.0.0"
```
Comment (kay0u) [inline manifest.json]: I don't see any helper that requires ynh >= 4.0, I think we should keep it compatible with stretch, especially for people having problems with php

**Review by kay0u — DISMISSED** [review]
Thank you for your work and patience

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,12 @@ Changelog
 ## Unreleased
 - Nothing for now...
 
+## [5.6~ynh1](https://github.com/YunoHost-Apps/wordpress_ynh/pull/115) - 2020-12-09
+
+#### Changed
+* [Update to 5.6](https://github.com/YunoHost-Apps/wordpress_ynh/pull/115/commits/2d72bf87c3e2e674e967058472586b31e0cb5c62)
+* [Use php-fpm.conf](https://github.com/YunoHost-Apps/wordpress_ynh/pull/115/commits/52b19005e83ad20a5a1bb11daa9ae22e945a2988)
```
Comment (kay0u) [inline CHANGELOG.md]:
```suggestion
```

**Review by kay0u — APPROVED** [review]
